app/components/post_body/index.js

```diff
@@ -30,10 +31,12 @@ const POST_TIMEOUT = 20000;
 
 function makeMapStateToProps() {
     const memoizeHasEmojisOnly = memoizeResult((message, customEmojis) => hasEmojisOnly(message, customEmojis));
+    const getReactionsForPost = makeGetReactionsForPost();
 
     return (state, ownProps) => {
         const post = ownProps.post;
         const channel = getChannel(state, post.channel_id) || {};
+        const reactions = getReactionsForPost(state, post.id);
```

Comment:
I'm curious if there are mobile-specific performance concerns with this change.

Reply:
Looks fine to me. Selector is a simple object selector and boolean comparison below does not cause re-renders. I am curious as to know why you are concerned.

Reply:
Heh, I'm still a little gun-shy about the memoization in our application and the negative impact to memory usage when overused, especially in a memory-constrained environment like the mobile app. Thanks for noting that you have no concerns :)